packages/ggit/src/output/output.ts

```diff
@@ -64,38 +66,23 @@ export interface InputValidationError {
  */
 export interface AllErrors {
   AllErrors: {
-    Errors: Error;
+    Errors: SoilMetricsError;
   };
 }
 
 /**
  *
  * @example
  */
-export interface Errors {
-  AllErrors: InputErrors;
-}
-/**
- *
- * @example
- */
-interface InputErrors {
-  InputValidationErrors: Error;
-}
-
-/**
- *
- * @example
- */
-interface Error {
-  ModelRun: ModelRunErrors;
+interface SoilMetricsError {
+  ModelRun: ModelRunErrors | ModelRunErrors[];
 }
```

Comment:
There was a duplicate definition of an interface named `Error`, so I renamed this to the un-imaginative, but unique name `SoilMetricsError`